targets/unity-v2/source/ExampleTestProject/Assets/PlayFabSDK/Client/PlayFabDeviceUtil.cs

```diff
@@ -118,10 +91,7 @@ private static void _OnPlayFabLogin(ClientModels.UserSettings settingsForUser, s
             }
 
             // Device attribution (adid or idfa)
-            if (settings.AdvertisingIdType != null && settings.AdvertisingIdValue != null)
-                DoAttributeInstall(settings, instanceApi);
-            else
-                GetAdvertIdFromUnity(settings, instanceApi);
+            GetAdvertIdFromUnity(settings, instanceApi);
```

Comment:
I'm 99% certain there's no need to call GetAdvertIdFromUnity anymore. It's the same system.
Remove the comment about Device attribution too.